css/main.css

```diff
@@ -52,26 +52,40 @@ footer svg {
   width: 1em;
 }
 
-#page-header {
-  color: white;
-  text-shadow: 4px 4px 1px rgb(13 66 163), -3px -3px 1px rgb(148 186 255);
-}
-
-#welcome-splash-button {
+button:not(.card-front):not(.card-back), select {
   background-color: var(--bg-color);
-  color:white;
+  color: white;
   border: 8px solid transparent;
-  max-height:37px;
+  max-height: 37px;
   border-radius: 5px;
   font-size: 1em;
   transition: background-color 0.3s ease;
   width: 200px;
   cursor: pointer;
 }
-#welcome-splash-button:hover {
+
+button:not(.card-front):not(.card-back):hover,
+select:hover {
   background-color: var(--accent-color);
 }
 
+#size-dropdown {
+  width: auto;
+}
+
+#page-header {
```

Comment:
While you're here, want to make the "The Regis Altare Birthday Project 2023" header text white as well?